tests/test-sources/plugins/bufferlines/bufferline.nix

```diff
@@ -1,7 +1,29 @@
+{ lib, pkgs, ... }:
 {
-  empty = {
-    plugins.bufferline.enable = true;
-  };
+  empty =
+    { config, ... }:
+    {
+      plugins.bufferline.enable = true;
+
+      assertions = [
```

Comment:
would it make more sense to test the package existence in our existing empty test cases?

Reply:
I think it's nice to keep the "empty" case simple.

Maybe we should consider adding `test.runNvim = false` to these assertion-based tests though, since we don't actually need to run neovim if all we care about is the assertion.

Reply:
Good idea, don't need to waste processing time that way if it's just evaluation time testing.

Reply:
Makes me think we could do with a `test.build` toggle to completely disable _building_ `finalPackage`, although we'd probably still want to ensure it is evaluated somehow (just not built)